kolibri/core/api.py

```diff
@@ -263,12 +263,12 @@ def serialize_object(self, **filter_kwargs):
         try:
             filter_kwargs = filter_kwargs or self._get_lookup_filter()
             return self.serialize(queryset.filter(**filter_kwargs))[0]
-        except IndexError:
+        
```

Comment:
The linter is complaining about this new blank line - if you remove it, it should be all good!

To avoid these issues in the future, please use pre-commit - see here for more details: https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#pre-commit-hooks